dm/tests/shardddl4_1/run.sh

```diff
@@ -975,23 +975,35 @@ function DM_155_CASE {
 	run_sql_source2 "insert into ${shardddl1}.${tb2} values(27,27,27);"
 
 	run_sql_source2 "alter table ${shardddl1}.${tb2} change c b int;"
-	sleep 1
+	sleep 15
```

Comment:
does this sleep is needed? check_log_contain_with_retry should contains 30 seconds retry

Reply:
i think we‘d better to sleep 15s. It may exceed 30s( maybe my local computor is slow) , but most of times, it's between 15s and 30s, or less than 15s.

Reply:
what's this sleep waiting for?

Reply:
if it's waiting for the handle ddl finished and check using `check_log_contain_with_retry`, maybe increase retry count, such as to 1 minute, in `check_log_contain_with_retry`, not add sleep here, to make it more flexible.

Reply:
yes, it is waiting for  the handle ddl.  add `check_log_contain_with_retry` to 1 minute will be more statable and it is beneficial for more ITs. i will fix.